cpr/session.cpp

```diff
@@ -286,6 +283,14 @@ void Session::Impl::SetLowSpeed(const LowSpeed& low_speed) {
     }
 }
 
+void Session::Impl::SetVerifySsl(const VerifySsl& verify) {
+    auto curl = curl_->handle;
+    if (curl) {
+        curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, verify ? 1L : 0L);
+        curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, verify ? 2L : 0L);
```

Comment:
Great work getting the defaults right. They're not very obvious unless you dig into the docs.